internal/controller/controllers/agent_controller.go

```diff
@@ -188,7 +188,32 @@ func (r *AgentReconciler) Reconcile(origCtx context.Context, req ctrl.Request) (
 	}
 
 	if agent.Spec.ClusterDeploymentName == nil && h.ClusterID != nil {
-		log.Debugf("ClusterDeploymentName is unset in Agent %s. unbind", agent.Name)
+		log.Debugf("ClusterDeploymentName is unset in Agent %s.", agent.Name)
+		installingStatuses := []string{
+			models.HostStatusPreparingForInstallation,
+			models.HostStatusPreparingFailed,
+			models.HostStatusPreparingSuccessful,
+			models.HostStatusInstalling,
+			models.HostStatusInstallingInProgress,
+			models.HostStatusInstallingPendingUserAction,
+		}
```

Comment:
Is this set something that we have elsewhere and can import?

Reply:
Yeah I think that makes sense.

Reply:
Cool! Went ahead and created an exported host installing statuses. Thanks!